dev-python/async_generator/async_generator-1.10.ebuild

```diff
@@ -1,24 +1,24 @@
-# Copyright 1999-2018 Gentoo Foundation
+# Copyright 1999-2018 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=6
+EAPI=7
```

Comment:
Hmmm, I guess if we don't want to revbump, we should keep EAPI=6.
We also shouldn't touch DOCS, right @mgorny?